.github/workflows/build.yml

```diff
@@ -305,7 +305,6 @@ jobs:
           krb5 \
           ossp-uuid \
           libxml2 \
-          git \
```

Comment:
Needs to be removed as the brew install conflicts with what the GitHub Action runner macOS environment already has installed and the job fails.